ai-platform-unified/notebooks/unofficial/pipelines/automl_tabular_classification_beans.ipynb

```diff
@@ -0,0 +1,1049 @@
+{
```

Comment:
<p>You might want to update to the cloud.google.com/vertex-ai link instead of cloud.google.com/ai-platform-unified</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/367/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
done